packager/media/codecs/h264_byte_to_unit_stream_converter.cc

```diff
@@ -52,6 +69,32 @@ bool H264ByteToUnitStreamConverter::GetDecoderConfigurationRecord(
   buffer.AppendInt(static_cast<uint16_t>(last_pps_.size()));
   buffer.AppendVector(last_pps_);
 
```

Comment:
Can you update the comment in line 36/37 and remove "last_sps_.size() + last_pps_.size() + 11" in line 38, which is no longer accurate?